apps/docs/src/development/theming.md

```diff
@@ -0,0 +1,20 @@
+# Theming Setup
+
+Onyx supports a dark and a light theme.  
+To learn more about the theming concept of onyx, take a look at our [colors documentation](/basics/colors.html)
+
+In order to let the user choose the preferred theme, we recommend [useDark of vueuse](https://vueuse.org/core/useDark/#usedark).  
```

Comment:
I would also document how the apply the dark mode without vueuse so something like:

"To apply the dark mode, just set the `.dark` class on your application (e.g. on the `<html>` element. For allowing the user to switch between light/dark mode, we recommend to use the `useDark` composable from vueuse"